src/axom/quest/SignedDistance.hpp

```diff
@@ -567,57 +567,55 @@ inline void SignedDistance<NDIMS, ExecSpace>::computeDistances(
   AXOM_UNUSED_VAR(result);
   SLIC_CHECK_MSG(result, "Input mesh is not an unstructured surface mesh");
 
-  AXOM_PERF_MARK_SECTION(
-    "ComputeDistances",
-    for_all<ExecSpace>(
-      npts,
-      AXOM_LAMBDA(std::int32_t idx) {
-        PointType qpt = queryPts[idx];
-
-        MinCandidate curr_min {};
-
-        auto searchMinDist = [&](std::int32_t current_node,
-                                 const std::int32_t* leaf_nodes) {
-          int candidate_idx = leaf_nodes[current_node];
-
-          checkCandidate(qpt,
-                         curr_min,
-                         candidate_idx,
-                         surfaceData,
-                         surf_pts,
-                         computeSigns);
-        };
-
-        auto traversePredicate = [&](const PointType& p,
-                                     const BoxType& bb) -> bool {
-          return axom::primal::squared_distance(p, bb) <= curr_min.minSqDist;
-        };
-
-        // Traverse the tree, searching for the point with minimum distance.
-        it.traverse_tree(qpt, searchMinDist, traversePredicate);
-
-        double sgn = 1.0;
-        if(computeSigns)
-        {
-          // STEP 0: if point is outside the bounding box of the surface mesh, then
-          // it is outside, just return 1.0
-          if(!(watertightInput && !boxDomain.contains(curr_min.minPt)))
-          {
-            sgn = computeSign(qpt, curr_min);
-          }
-        }
+  AXOM_ANNOTATE_SCOPE("ComputeDistances");
```

Comment:
Please consider moving `AXOM_ANNOTATE_SCOPE` to the open-brace, about 30 lines prior.

Reply:
Sure.

Note that the requested change would add increase the timing somewhat since it'll include a bit more setup code, which I think should be ok.